src/Arcus.Security.Tests.Runtimes.AzureFunctions/Arcus.Security.Tests.Runtimes.AzureFunctions.csproj

```diff
@@ -1,11 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
+    <OutputType>Exe</OutputType>
```

Comment:
I wonder what happens if you remove the <outputtype> property ?

Reply:
`OutputType`? This is required for isolated Azure Functions.
No documentation on this available, though; and obscure error messages when it's missing, a.k.a. Microsoft at it best 💯.